.travis.yml

```diff
@@ -111,7 +111,7 @@ matrix:
       name: Test with PHP nightly (all drivers)
       php: nightly
       env:
-        - IMAGINE_DRIVER: all
+        - IMAGINE_DRIVER: imagick
```

Comment:
This should be reverted once the GMagick issue has been resolved, <https://bugs.php.net/bug.php?id=80106>